core/templates/dev/head/pages/exploration-editor-page/services/exploration-states.service.spec.ts

```diff
@@ -144,4 +148,21 @@ describe('ExplorationStatesService', function() {
       });
     });
   });
+
+
+  it('solicit answer details value should be false', function() {
+    expect(
+      ExplorationStatesService.getSolicitAnswerDetailsMemento(
+        'Hola', 'solicit_answer_details')).toEqual(false);
+  });
+
+  it('save solicit answer details to true', function() {
```

Comment:
I think getter and setter can be tested via a single test. So just concentrate on writing one good test. (Ping me if you need any help.)

Reply:
Done